x-pack/plugins/event_log/server/lib/ready_signal.test.ts

```diff
@@ -27,7 +27,6 @@ describe('ReadySignal', () => {
 
       const innerValue = await readySignal.wait();
       expect(innerValue).toBe(42);
-      done();
```

Comment:
This test has other issues, like errors from the expect() in this function not bubbling up appropriately, but I'm fairly confident that with this change the test is going to exit successfully before this assertion runs (after the 400ms timer is up).